addons/dialogue_nodes/nodes/sub_nodes/DefaultForkOption.tscn

```diff
@@ -0,0 +1,9 @@
+[gd_scene load_steps=2 format=3 uid="uid://c0u5fek751c5f"]
```

Comment:
This scene could just be a ForkOption instance in ForkNode.tscn with its `editable` property set to false.

Reply:
Yeah, I don't like using the _Editable Children_ feature in general as it has brought me considerable headaches in the past, but we can skip the extra .tscn and just make an edited child.

Reply:
Or the `editable` property could be an `@export var` instead?

Reply:
I think it's fine. The "Default" option is just a regular option that you should not be able to write on. The change is so small, simple, and permanent I don't feel is necessary to add extra code just for this. It's fine being an edited child imo.